libraries/APM_Control/AR_PosControl.cpp

```diff
@@ -157,12 +157,16 @@ void AR_PosControl::update(float dt)
         _vel_target.y = vel_3d_cms.y * 0.01;
     }
 
-    // calculate desired acceleration
-    // To-Do: fixup _limit_vel used below
-    _accel_target = _pid_vel.update_all(_vel_target, curr_vel_NED.xy(), dt, _limit_vel);
+    // calculate desired acceleration.  I-term build up disabled in forward-back direction
+    const Vector2f limit_vel = AP::ahrs().body_to_earth2D(Vector2f{1, 0});
```

Comment:
I think this limit only works if the is going too slowly, but not if its going too fast, it should be `-1, 0` in that case. To be honest I don't think it really matters, since your setting the integrator directly on 169.

Reply:
This should be removed as it prevent build up if any component is in the forward direction.

Reply:
OK, thanks.  After a discussion with Leonard I've replaced the limit vector with one that is set based on new attitude control steering limit flags.